parlai/mturk/tasks/model_evaluator/worlds.py

```diff
@@ -48,3 +48,10 @@ def report(self):
     def shutdown(self):
         self.task_world.shutdown()
         self.mturk_agent.shutdown()
+        
+    def review_work(self):
```

Comment:
maybe add something like
```
"""Programmatically approve/reject the turker's work.
For example:
.. code-block:: python
    if self.turkerResponse == '0':
        self.mturk_agent.reject_work('You rated our model's response as a 0/10 but we know we're better than that')
    else:
        if self.turkerResponse == '10':
            self.mturk_agent.payBonus(1, 'Thanks for the great rating!')
        self.mturk_agent.approve_work()
"""
```

maybe we should have a parent class for these mturk worlds so that we have a better place to put documentation that applies to all tasks, like this